### DISCUSSION TITLE
请问关闭外网访问的方式？

### DISCUSSION THREAD

**Discussion by BackMountainDevil** [PBH-BTN/PeerBanHelper#358] [问答] [answered]
成功配置了qb和peerbanhelper。考虑了安全，测试了一下内外网访问，发现可以通过公网v6访问peerhelper，qb则不行，出于安全考虑，我想关闭这个外网访问，但是没发现软件里有这个配置项

**Comment by Ghost-chu** [comment]
将监听地址改成 127.0.0.1 就行了

**Comment by BackMountainDevil** [comment]
我在webui的设置里把endpoint设置为 http://localhost:9898，测试内网ipv4和公网v6地址都可以访问掉，重启过pbh还是一样可以被访问到唉，请问还需要在哪里修改监听地址呢？





------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年8月22日(星期四) 凌晨0:02
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [PBH-BTN/PeerBanHelper] 请问关闭外网访问的方式？ (Discussion #358)

**Reply by Ghost-chu** [reply]
[address](https://github.com/PBH-BTN/PeerBanHelper/blob/2d3b3bc6632666416ece539e2a5f5b1b49ac4f24/src/main/resources/config.yml#L17-L19)

**Comment by BackMountainDevil** [comment, marked as answer]
感动全球，本地配件文件地址“PeerBanHelper\data\config\config.yml”，server.address 由 0.0.0.0 修改为 127.0.0.1





------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年8月25日(星期天) 下午5:10
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [PBH-BTN/PeerBanHelper] 请问关闭外网访问的方式？ (Discussion #358)












address

**Reply by johndavid1** [reply]
请问一下，新版本data下无config从哪儿改呢

**Reply by CreeperAWA** [reply]
> 请问一下，新版本data下无config从哪儿改呢

Windows 的话从这里可以直接打开对应目录
<img width="1902" height="1071" alt="image" src="https://github.com/user-attachments/assets/1f832c61-4658-4ad9-8b04-23db7ea44f6b" />
